pandora-client-web/src/components/chatroomSelect/chatroomSelect.tsx

```diff
@@ -25,9 +26,10 @@ const TIPS: readonly string[] = [
 	`Your character can turn around for everyone in a chat room in the "Pose" tab or with "/turn".`,
 	`Chat commands start with a "/" and typing just this one character shows a help menu.`,
 	`You can use your browser's "back" and "forward" buttons to navigate between screens.`,
-	`The dragging points to move room devices are invisible, but generally under the item.`,
 	`In the Pandora settings, character (chat) and account (direct messages) name colors are set separately.`,
 	`Every single change in the wardrobe happens instantly and is immediately visible to everyone in the room.`,
+	`You need to equip the handheld item "room construction tools" to decorate your rooms`,
```

Comment:
`to decorate your rooms` could be changed to `to setup devices` or something similar that is a bit more clear